Runtime/Scripts/ColyseusRequest.cs

```
@@ -58,19 +58,25 @@ public async Task<string> Request(string uriMethod, string uriPath, string uriQu
                 {
                     if (_serverSettings.useSecureProtocol)
                     {
-                         //We failed to make this call with a secure protocol, try with non-secure and if that works we'll stick with it
-                         _serverSettings.useSecureProtocol = false;
-                         LSLog.LogError($"Failed to make request to {req.url} with secure protocol, trying again without!");
-                         return await Request(uriMethod, uriPath, uriQuery, Token, data);
+                         //We failed to make this call with a secure protocol, try again
+                         LSLog.LogError($"Failed to make request to {req.url} with secure protocol, trying again!");
+                         try
+                         {
+	                         return await Request(uriMethod, uriPath, uriQuery, Token, data);
+                         }
+                         catch
+                         {
+	                         LSLog.LogError($"Failed RETRY to make request to {req.url}.");
+                         }
```

I think you want to `throw` here after logging, otherwise it eats any exceptions, whereas before it did not.

---

I actually do want it to eat the exception, bc we will retry on the client and catch exception there. Currently we get 2 exceptions if this fails; one from here and one from `ServerManager.cs` on the client

---

:+1: